Add set_guest_wifi_password description

sti0 opened the pull request:
Update docs for PR https://github.com/home-assistant/core/pull/62892

## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
- Add documentation for new service `set_guest_wifi_password` (https://github.com/home-assistant/core/pull/62892)


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/62892
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Commit:
Add description for set_guest_wifi_password service

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.

mib1185 commented on source/_integrations/fritz.markdown:
```diff
@@ -59,6 +59,18 @@ A device is identified as stale when it's still present on Home Assistant but no
+| ---------------------- | -------- | -------------------------------------------------------------------------------------------------------------- |
+| `device_id`            | no       | Only act on a specific  router                                                                                 |
+| `password`             | yes      | New password for the guest wifi                                                                                |
+| `length`               | yes      | Length of the new password. The password will be auto-generated, if no password is set.                        |
```
```suggestion
| `length`               | yes      | Length of the auto-generated password. (_default 12_)                        |
```

sti0 committed:
Update default length

Instead of the 32 characters we now use the upstreams library's default value which is 12 chars.

mib1185 commented on source/_integrations/fritz.markdown:
```diff
@@ -59,6 +59,18 @@ A device is identified as stale when it's still present on Home Assistant but no
+
+Set a new password for the guest wifi.
+The password must be between 8 and 63 characters long.
+If no additional parameter is set, the password will be auto-generated with a length of 12 characters.
```
```suggestion
If no password is given, it will be auto-generated.
```

sti0 committed:
Better description for `length`

Co-authored-by: Michael <35783820+mib1185@users.noreply.github.com>

sti0 committed:
Simpler description when no password is given

Co-authored-by: Michael <35783820+mib1185@users.noreply.github.com>
